README.md

```diff
@@ -161,6 +161,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [Snipplr](https://snipplr.com/all) - Code snippet search engine that allows users to search and share code snippets across various programming languages and frameworks
 - [Postman Public Collections](https://www.postman.com/explore/collections) - Explore the best APIs, collections, workspaces in the world on the Postman Public API Network
 - [ScriptMafia](https://scriptmafia.org) - Download full nulled scripts
+- [Searchftps](https://www.searchftps.net/) - Online tool for searching and accessing files on FTP servers
```

Comment:
Delete this, already handled by https://github.com/edoardottt/awesome-hacker-search-engines/pull/166